docs-v2/source/includes/_tipser-elements.md

```diff
@@ -367,6 +367,10 @@ Displays the price, discount price and unit price ( eg. 2$/100ml, if applicable
 
 Displays the full-size version of the active product image. With some configuration options it can be also used to change the active product image.
 
+<aside class="notice">
+ProductImage component requires an additional container with fixed width and height.
```

Comment:
ProductImage component requires an additional container with fixed width and height. -> ProductImage component must be placed in a container with fixed width and height.

Poza tym opakuj nazwę product image w tagi code, żeby wyświetliła się jako nazwa kodowa